kitsune/sumo/views.py

```diff
@@ -105,7 +105,12 @@ def robots(request):
     if not settings.ENGAGE_ROBOTS:
         template = "User-Agent: *\nDisallow: /"
     else:
-        template = render(request, "sumo/robots.html")
+        if not request.build_absolute_uri("/").startswith("https://support.mozilla.org"):
```

Comment:
I would recommend using `request.get_host()` instead of `request.build_absolute_uri("/")`, since we only care about the requested host.

Reply:
Ah, thanks.